src/submission/submission-to-clinical/merge-submission.ts

```diff
@@ -116,8 +129,45 @@ const updateSpecimenInfo = (donor: Donor, record: any) => {
   return specimen;
 };
 
-const addOrUpdateTreatementInfo = (donor: Donor, record: any): Treatment => {
-  const submitterTreatementId = record[TreatmentFieldsEnum.submitter_treatment_id];
+const updateFollowUp = (donor: Donor, record: ClinicalEnitityRecord) => {
+  const followUp = findFollowUp(
+    donor,
+    record[ClinicalUniqueIndentifier[ClinicalEntitySchemaNames.FOLLOW_UP]],
+  );
+
+  if (followUp) {
+    followUp.clinicalInfo = record;
+    return;
+  }
+
+  if (!donor.followUps) {
+    donor.followUps = [];
+  }
+
+  donor.followUps.push({
+    clinicalInfo: record,
+  });
+};
+
+const addOrUpdateFollowUpInfoInDonor = (donor: Donor, record: ClinicalInfo) => {
```

Comment:
some thing here doesn't look right, this method is setting treatments if followUp doesn't exist?? Looks like copy pasta gone wrong. I think you meant to use the above function...

Reply:
Correct will fix